### PR TITLE
DOCS-6410: Added Troubleshooting Guide

### PR DESCRIPTION
**Pull request by acurci-at-mulesoft**
Added Troubleshooting Guide
Fixed version number

### CODE REVIEW

File: jms/1.8/modules/ROOT/pages/jms-troubleshooting.adoc [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+To resolve this error:
+
+Add the driver dependency at the domain level, which enables application redeployment.
+
+== HA Enviroments and Durable Suscriptions
+
+The configuration of HA systems consists of two components: the primary node (which is providing continuous service) and the secondary node (which is a clone of the previous one so that it provides the same service). Both of them must be identical so that the end-user is not aware of the system crash and can continue to use it transparently.
+When you are working with Durable Suscriptions, only one node can getting the connection and the other one fails.
+In this kind or scenarios, the nodes are not clones since one of them has something that the other does not: the connection.
+The possibility of using connections that cannot be initialized in multiple nodes simultaneously was not considered in the design of Mule running in HA clusters. The current design is aimed at having all nodes initializing connections to ensure that they are capable of facing a failure of the primary node.
```
Comment (nicomz) [inline jms/1.8/modules/ROOT/pages/jms-troubleshooting.adoc]: I'd reword this. Saying that a feature was not considered give's me a negative sentiment.

File: jms/1.8/modules/ROOT/pages/jms-troubleshooting.adoc [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+The configuration of HA (High Availability) systems consists of two components:
+
+* Primary node +
+Is the main node that provides continuous service.
+* Secondary node +
+Is a clone of the primary node and provides the same service.
+
+Both nodes must be identical so the end-user is not aware of the system crash and can continue to use the system transparently.
+
+However, when you work with durable subscriptions, only one node initialize a connection while the other node fails to do it. In these scenarios, the nodes are not clones because one node has a connection while the other one doesn't. The current design of Mule running in HA clusters aims that all nodes initialize connections to ensure that they are capable of facing a failure of the primary node.
```
Comment (sathya0) [inline jms/1.8/modules/ROOT/pages/jms-troubleshooting.adoc]:
```suggestion
However, when you work with durable subscriptions, only one node initializes a connection while the other node fails to do it. In these scenarios, the nodes are not clones because one node has a connection while the other one doesn't. The current design of Mule running in HA clusters aims that all nodes initialize connections to ensure that they are capable of facing a failure of the primary node.
```

**Review by sathya0 — DISMISSED** [review]

**Review by sathya0 — DISMISSED** [review]

**Review by sathya0 — APPROVED** [review]
